consensus/consensus.go

```diff
@@ -73,29 +65,21 @@ type Consensus struct {
 	viewIDBitmap map[uint64]*bls_cosi.Mask
 	m1Payload    []byte     // message payload for type m1 := |vcBlockHash|prepared_agg_sigs|prepared_bitmap|, new leader only need one
 	vcLock       sync.Mutex // mutex for view change
-
 	// The chain reader for the blockchain this consensus is working on
 	ChainReader *core.BlockChain
-
 	// map of nodeID to validator Peer object
 	validators sync.Map // key is the hex string of the blsKey, value is p2p.Peer
-
 	// Minimal number of peers in the shard
 	// If the number of validators is less than minPeers, the consensus won't start
-	MinPeers int
-
+	MinPeers   int
```

Comment:
what's the change here?

Reply:
whitespace and remove this unused field, uniqueIDInstance *utils.UniqueValidatorID